### PR TITLE
Adds a local cluster connector so that development functions

### PR DESCRIPTION
**Pull request by tso**

### CODE REVIEW

File: misk/src/main/kotlin/misk/eventrouter/RealEventRouterModule.kt [hunk 1]
```diff
@@ -18,10 +19,15 @@ import javax.inject.Singleton
 
 class RealEventRouterModule : KAbstractModule() {
   override fun configure() {
+    val environment = Environment.fromEnvironmentVariable()
```
Comment (tso) [inline misk/src/main/kotlin/misk/eventrouter/RealEventRouterModule.kt]: This should be `@Inject lateinit var environment: Environment`

File: misk/src/main/kotlin/misk/eventrouter/LocalClusterConnector.kt [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package misk.eventrouter
+
+import misk.web.actions.WebSocket
+import misk.web.actions.WebSocketListener
+import javax.inject.Singleton
+
+@Singleton
+internal class LocalClusterConnector : ClusterConnector {
+  override fun joinCluster(topicPeer: TopicPeer) {
+    topicPeer.clusterChanged(ClusterSnapshot(listOf("local"), "local"))
```
Comment (tso) [inline misk/src/main/kotlin/misk/eventrouter/LocalClusterConnector.kt]: Should use the kubernetes config to get `my_pod_name`

File: exemplarchat/src/main/resources/chat-development.yaml [hunk 3]
```diff
@@ -1,3 +1,3 @@
 kubernetes:
-  my_pod_namespace: none
-  my_pod_name: none
+  my_pod_namespace: default
+  my_pod_name: Localhost
```
Comment (swankjesse) [inline exemplarchat/src/main/resources/chat-development.yaml]: should this be lowercase? I’ve never seen the string `Localhost` with a `L` and it looks so weird to my eyes

File: misk/src/main/kotlin/misk/eventrouter/LocalClusterConnector.kt (hunk 2 above)
Comment (swankjesse) [reply misk/src/main/kotlin/misk/eventrouter/LocalClusterConnector.kt]: `local` or `Localhost` ? Are these synonyms?

File: misk/src/main/kotlin/misk/eventrouter/LocalClusterConnector.kt [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  }
+
+  override fun leaveCluster(topicPeer: TopicPeer) {
+  }
+
+  override fun connectSocket(hostname: String, listener: WebSocketListener): WebSocket {
+    throw IllegalStateException("In a single machine system there should be no network requests")
```
Comment (swankjesse) [inline misk/src/main/kotlin/misk/eventrouter/LocalClusterConnector.kt]: nice

File: misk/src/main/kotlin/misk/eventrouter/RealEventRouterModule.kt (hunk 1 above)
Comment (swankjesse) [reply misk/src/main/kotlin/misk/eventrouter/RealEventRouterModule.kt]: ahh, you can’t ’cause this is itself a module. Pass the environment to the module constructor?

**Pull request merged**